release/api/v1alpha1/artifact_types.go

```diff
@@ -83,3 +83,9 @@ type Manifest struct {
 	// URI points to the manifest yaml file
 	URI string `json:"uri,omitempty"`
 }
+
+type HelmRegistry struct {
```

Comment:
Wouldn't it be easier to only support one registry for container images and helm charts?